DiscordBot/Commands/TicTacToeCommand.cs

```diff
@@ -0,0 +1,191 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using Microsoft.CSharp.RuntimeBinder;
+using TicTacToe;
+using static TicTacToe.SquareState;
```

Comment:
Don't do `using static`, it introduces confusing code.
Not recommended in general.

Reply:
ok